ModAssistant/Localisation/nl.xaml

```diff
@@ -19,6 +19,7 @@
     <sys:String x:Key="MainWindow:ModsButton">Mods</sys:String>
     <sys:String x:Key="MainWindow:AboutButton">Over</sys:String>
     <sys:String x:Key="MainWindow:OptionsButton">Opties</sys:String>
+    <sys:String x:Key="MainWindow:PlayButton">Speel</sys:String>
```

Comment:
You have added an exclamation mark in every other language, so i've added it here to stay consistent.
```suggestion
    <sys:String x:Key="MainWindow:PlayButton">Speel!</sys:String>
```